SteamTransport/Util.cs

```diff
@@ -0,0 +1,173 @@
+				IntPtr.Zero,
+				ESteamNetworkingConfigDataType.k_ESteamNetworkingConfig_Float,
+				floatHandle.AddrOfPinnedObject()
+			);
+			SteamNetworkingUtils.SetConfigValue(
+				ESteamNetworkingConfigValue.k_ESteamNetworkingConfig_FakePacketLoss_Recv,
+				ESteamNetworkingConfigScope.k_ESteamNetworkingConfig_Global,
+				IntPtr.Zero,
+				ESteamNetworkingConfigDataType.k_ESteamNetworkingConfig_Float,
+				floatHandle.AddrOfPinnedObject()
+			);
+
+			SteamNetworkingUtils.SetConfigValue(
+				ESteamNetworkingConfigValue.k_ESteamNetworkingConfig_FakePacketLag_Send,
+				ESteamNetworkingConfigScope.k_ESteamNetworkingConfig_Global,
+				IntPtr.Zero,
+				ESteamNetworkingConfigDataType.k_ESteamNetworkingConfig_Int32,
+				intHandle.AddrOfPinnedObject()
+			);
+			SteamNetworkingUtils.SetConfigValue(
+				ESteamNetworkingConfigValue.k_ESteamNetworkingConfig_FakePacketLag_Recv,
+				ESteamNetworkingConfigScope.k_ESteamNetworkingConfig_Global,
+				IntPtr.Zero,
+				ESteamNetworkingConfigDataType.k_ESteamNetworkingConfig_Int32,
+				intHandle.AddrOfPinnedObject()
+			);
+
+			SteamNetworkingUtils.SetConfigValue(
+				ESteamNetworkingConfigValue.k_ESteamNetworkingConfig_FakePacketReorder_Send,
+				ESteamNetworkingConfigScope.k_ESteamNetworkingConfig_Global,
+				IntPtr.Zero,
+				ESteamNetworkingConfigDataType.k_ESteamNetworkingConfig_Float,
+				floatHandle.AddrOfPinnedObject()
+			);
+			SteamNetworkingUtils.SetConfigValue(
+				ESteamNetworkingConfigValue.k_ESteamNetworkingConfig_FakePacketReorder_Recv,
+				ESteamNetworkingConfigScope.k_ESteamNetworkingConfig_Global,
+				IntPtr.Zero,
+				ESteamNetworkingConfigDataType.k_ESteamNetworkingConfig_Float,
+				floatHandle.AddrOfPinnedObject()
+			);
+			SteamNetworkingUtils.SetConfigValue(
+				ESteamNetworkingConfigValue.k_ESteamNetworkingConfig_FakePacketReorder_Time,
+				ESteamNetworkingConfigScope.k_ESteamNetworkingConfig_Global,
+				IntPtr.Zero,
+				ESteamNetworkingConfigDataType.k_ESteamNetworkingConfig_Int32,
+				intHandle.AddrOfPinnedObject()
+			);
+
+			SteamNetworkingUtils.SetConfigValue(
+				ESteamNetworkingConfigValue.k_ESteamNetworkingConfig_FakePacketDup_Send,
+				ESteamNetworkingConfigScope.k_ESteamNetworkingConfig_Global,
+				IntPtr.Zero,
+				ESteamNetworkingConfigDataType.k_ESteamNetworkingConfig_Float,
+				floatHandle.AddrOfPinnedObject()
+			);
+			SteamNetworkingUtils.SetConfigValue(
+				ESteamNetworkingConfigValue.k_ESteamNetworkingConfig_FakePacketDup_Recv,
+				ESteamNetworkingConfigScope.k_ESteamNetworkingConfig_Global,
+				IntPtr.Zero,
+				ESteamNetworkingConfigDataType.k_ESteamNetworkingConfig_Float,
+				floatHandle.AddrOfPinnedObject()
+			);
+			SteamNetworkingUtils.SetConfigValue(
+				ESteamNetworkingConfigValue.k_ESteamNetworkingConfig_FakePacketDup_TimeMax,
+				ESteamNetworkingConfigScope.k_ESteamNetworkingConfig_Global,
+				IntPtr.Zero,
+				ESteamNetworkingConfigDataType.k_ESteamNetworkingConfig_Int32,
+				intHandle.AddrOfPinnedObject()
+			);
```

Comment:
lot of repeated code - seperate out to local function that just takes `ESteamNetworkingConfigValue`, `ESteamNetworkingConfigDataType`, and value?